metatests.js

```diff
@@ -23,3 +23,19 @@ module.exports = {
   testAsync,
   walk,
 };
+
+if (process.browser) {
+  process.exit = code => {
```

Comment:
This https://github.com/metarhia/metatests/pull/79 will allow you to not override this function I think.